shared-helpers/src/programHelpers.ts

```diff
@@ -14,20 +14,30 @@ export const mapProgramToApi = (program: Program, data: Record<string, any>) =>
   const [key, value] = Object.entries(data)[0]
   const options = []
 
-  options.push({
-    key: value,
-    checked: true,
-    extraData: [],
-  })
-  program?.formMetadata?.options.forEach((option) => {
-    if (option.key !== value) {
+  if (program?.formMetadata?.type === "checkbox") {
```

Comment:
Please update this to use the new enum, e.i `program?.formMetadata?.type === FormMetaDataType.checkbox`